test/run-spec-wasm2c.py

```diff
@@ -445,29 +445,32 @@ def _Action(self, command):
             raise Error('Unexpected action type: %s' % type_)
 
 
-def Compile(cc, c_filename, out_dir, *cflags):
+def Compile(cc, c_filename, out_dir, use_c11, *cflags):
     if IS_WINDOWS:
         ext = '.obj'
     else:
         ext = '.o'
     o_filename = utils.ChangeDir(utils.ChangeExt(c_filename, ext), out_dir)
     args = list(cflags)
+
     if IS_WINDOWS:
-        args += ['/nologo', '/MDd', '/c', c_filename, '/Fo' + o_filename]
+        cstd_flag = ['/std:c11', '/experimental:c11atomics'] if use_c11 else []
+        args += cstd_flag + ['/nologo', '/MDd', '/c', c_filename, '/Fo' + o_filename]
     else:
         # See "Compiling the wasm2c output" section of wasm2c/README.md
         # When compiling with -O2, GCC and clang require '-fno-optimize-sibling-calls'
         # and '-frounding-math' to maintain conformance with the spec tests
         # (GCC also requires '-fsignaling-nans')
-        args += ['-c', c_filename, '-o', o_filename, '-O2',
-                 '-Wall', '-Werror', '-Wno-unused',
-                 '-Wno-array-bounds',
-                 '-Wno-ignored-optimization-argument',
-                 '-Wno-tautological-constant-out-of-range-compare',
-                 '-Wno-infinite-recursion',
-                 '-fno-optimize-sibling-calls',
-                 '-frounding-math', '-fsignaling-nans',
-                 '-std=c99', '-D_DEFAULT_SOURCE']
+        cstd_flag = ['-std=c11'] if use_c11 else ['-std=c99']
+        args += cstd_flag + ['-c', c_filename, '-o', o_filename, '-O2',
+                             '-Wall', '-Werror', '-Wno-unused',
+                             '-Wno-array-bounds',
+                             '-Wno-ignored-optimization-argument',
+                             '-Wno-tautological-constant-out-of-range-compare',
+                             '-Wno-infinite-recursion',
+                             '-fno-optimize-sibling-calls',
+                             '-frounding-math', '-fsignaling-nans',
+                             '-D_DEFAULT_SOURCE']
```

Comment:
Nit: How about doing this below which avoid the reindenting and also avoids the single line `if/`else` which I'm not a fan of myself.

```
if use_c11:
  args.append('-std=c99')
else:
  args.append('-std=c11')
```

Reply:
will fix